app/models/line_item.rb

```diff
@@ -16,5 +16,5 @@ class LineItem < ApplicationRecord
   belongs_to :item
 
   validates :item_id, presence: true
-  validates :quantity, numericality: { greater_than: 0 }
+  validates :quantity, numericality: { other_than: 0, only_integer: true }
```

Comment:
Do you need only_integer if you already have `numericality` and the db schema specifies it as an integer? (I genuinely don't know.)

Reply:
I don't want you to look it up, I'm very happy with this way, I'm just curious because I probably wouldn't have used the `only_integer` validator.

Reply:
According to the API if you dont' explicitly say `only_integer`, it allows floats.  

http://guides.rubyonrails.org/active_record_validations.html#numericality